packages/react-components/src/types/VideoGalleryParticipant.ts

```diff
@@ -90,4 +90,10 @@ export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
   isSpeaking?: boolean;
   /** Video stream of shared screen */
   screenShareStream?: VideoGalleryStream;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** State that the participant is in */
+  state?: string;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** the type of participant that it is*/
+  kind?: 'communicationUser' | 'phoneNumber' | 'microsoftTeamsUser' | 'unknown';
```

Comment:
Where are we planning to use this prop?

Reply:
the kind? this would be used either in the `VideoGallery` itself to give a PSTN participant tile or in the `RemoteVideoTile` to change the UI based on what kind of user it is.